src/Facebook/InstantArticles/Transformer/Transformer.php

```diff
@@ -97,82 +91,42 @@ public static function markAsProcessed($node)
      *
      * @param DOMNode $node The node to clone
```

Comment:
Outdated docs as well, let's change it to:
```php
/**
 * @param DOMNode The node of interest
 * @return bool Whether the node is processed or not
 */
```

Reply:
Fixed.